views/pages/elections/elections.pug

```diff
@@ -1,72 +1,82 @@
+
+      if currentElection.decisions.length > 0
+        +banner-image("/assets/img/banners/pinktie-mc.jpeg", ['pink'], 'Allegations and Appeals')
+        +section(null,"white")
+          h2 Decisions 
+          - 
+            const decisions = currentElection.decisions.map((decision, index) => {
+              decision.index = index;
+              return decision;
+            })
+          ul 
+            each decision of decisions 
+              a(href="#decision-" + decision.index)
+                li #{decision.candidate}
           each decision of decisions 
-            a(href="#decision-" + decision.index)
-              li #{decision.candidate}
-        each decision of decisions 
-          div(class="elections-decision" id="decision-" + decision.index)
-            h2 Against #{decision.candidate}
-            span #[strong Submitted:] #{decision.date}
-            div(class="decision-body")
-              h3 Summary of allegation:
-              p!=decision.allegationMarkdown
-              h4 Counterevidence and defence:
-              p!=decision.defenseMarkdown
-              h4 Decision: 
-              p!=decision.decisionMarkdown
-              if(decision.penalties)
-                h3 Penalty: 
-                ul(class="penalties")
-                  for penalty of decision.penalties
-                    li=penalty
-                if(decision.penaltyDescription) 
-                  p!=decision.penaltyDescriptionMarkdown
-                if(decision.appeal)
-                  h4 Appeal 
-                  p!=decision.appealMarkdown
-                  h4 Decision:
-                  p!=decision.appealDecisionMarkdown
+            div(class="elections-decision" id="decision-" + decision.index)
+              h2 Against #{decision.candidate}
+              span #[strong Submitted:] #{decision.date}
+              div(class="decision-body")
+                h3 Summary of allegation:
+                p!=decision.allegationMarkdown
+                h4 Counterevidence and defence:
+                p!=decision.defenseMarkdown
+                h4 Decision: 
+                p!=decision.decisionMarkdown
+                if(decision.penalties)
+                  h3 Penalty: 
+                  ul(class="penalties")
+                    for penalty of decision.penalties
+                      li=penalty
+                  if(decision.penaltyDescription) 
+                    p!=decision.penaltyDescriptionMarkdown
+                  if(decision.appeal)
+                    h4 Appeal 
+                    p!=decision.appealMarkdown
+                    h4 Decision:
+                    p!=decision.appealDecisionMarkdown
+  else
+    +banner-image("/assets/img/banners/mathsoc-wall.jpeg", ["dark-grey"], `Elections`)
+    div(class="content")
+      +section(null,"white")
+        div(class="elections-unavailable")
+          p Election information is currently unavailable. Please check back later.
```

Comment:
Can you make this text be drawn from the `/data` files? It would allow future execs to have a lot more customizability!

Reply:
Hey River,
Hoping I'm understanding this correctly, but in my latest commit, the page pulls data under the assumption that the elections.json schema is modified like this:

`
{
    "noElectionsMessage": "Election information is currently unavailable. Please check back later.",
    "electionsData": []
}
`

Since the /data files are not tracked by the repo, wasn't sure if I should be documenting this change anywhere, and/or if this is the solution you were looking for. Please let me know!

Reply:
Hey! Good catch - can you update `/data-base` and `schemas.ts` (i think that's the file name) with the new schema?

Reply:
Done!